mypy/modulefinder.py

```diff
@@ -722,6 +723,11 @@ def compute_search_paths(sources: List[BuildSource],
     # Start with a MYPYPATH environment variable at the front of the mypy_path, if defined.
     mypypath = mypy_path()
 
+    # Push the non-default mypy prefix site-packages at the back, if any
+    if options._prefix is not None:
+        prefix_sitepkgs, = site.getsitepackages([str(options._prefix.resolve())])
```

Comment:
`site.getsitepackages` doesn't take any arguments, perhaps you mean to do something else here?

Reply:
I think the documentation needs to be improved:
https://github.com/python/cpython/blob/e06ae75e16dbf46b6626b6a41b62391ea1fdb319/Lib/site.py#L346

Here I'm using the optional `prefixes` argument to get the sitepackages path for the detected mypy prefix, rather than the default ones.